robottelo/products.py

```diff
@@ -933,6 +934,12 @@ def setup_virtual_machine(
                 repo['label'] for repo in self.custom_repos_info if repo['content-type'] == 'yum'
             ]
 
+        if location:
+            vm.execute(
```

Comment:
@JacobCallahan Can you think of a more elegant way to do such a thing using the broker library?

Reply:
Let me know if this helps :) 

https://github.com/SatelliteQE/robottelo/blob/f7ed1b80c2778b56183e7fbe23cb9f182fa59741/robottelo/host_helpers/contenthost_mixins.py#L103

Reply:
Implemented, thanks! I seem to be the first user of that method.